Sources/DeepLearning/Layer.swift

```diff
@@ -1303,8 +1303,10 @@ public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
     }
 
     public typealias State = Tensor<Scalar>
-    public typealias Input = RNNCellInput<Tensor<Scalar>, State>
-    public typealias Output = RNNCellOutput<State, State>
+    public typealias TimeStepInput = Tensor<Float>
```

Comment:
`Float` -> `Scalar`

Reply:
Ah sorry, completely missed that. Fixing now.